src/components/editor-page/editor-pane/autocompletion/code-block.ts

```diff
@@ -7,9 +7,11 @@
 import { Editor, Hint, Hints, Pos } from 'codemirror'
 import { findWordAtCursor, generateHintListByPrefix, Hinter } from './index'
 import { showErrorNotification } from '../../../../redux/ui-notifications/methods'
+import { Logger } from '../../../../utils/logger'
 
 type highlightJsImport = typeof import('../../../common/hljs/hljs')
 
+const log = new Logger('autocompletion code')
```

Comment:
_Autocompletion code_ sounds a bit strange to me.
Maybe write codeblocks instead of code?